helpers/json-schema/validator.js

```diff
@@ -38,6 +49,15 @@ JsonSchema.options = utils.extend({}, Validator.options);
 JsonSchema.validate = Validator.validate;
 
 JsonSchema.addRule('type', {
+	/**
+	 * Description of the function.
+	 *
+	 * @func type_validate
```

Comment:
Is this kind of underscoring standard? It looks strange with camel case below.

Reply:
Where do you see camelCase?

Reply:
Where do you see camelCase?

Reply:
Lower down, I can't remember the variable name but it looks like this: `@func camelCase_validate`.